pkg/cmd/server/apikey/generate.go

```diff
@@ -13,13 +13,14 @@ import (
 	"github.com/daytonaio/daytona/cmd/daytona/config"
 	"github.com/daytonaio/daytona/internal/util/apiclient"
 	"github.com/daytonaio/daytona/internal/util/apiclient/server"
+	"github.com/daytonaio/daytona/pkg/views"
 	"github.com/daytonaio/daytona/pkg/views/server/apikey"
-	"github.com/daytonaio/daytona/pkg/views/util"
+	view "github.com/daytonaio/daytona/pkg/views/server/apikey"
 )
 
 var saveFlag bool
 
-var generateCmd = &cobra.Command{
+var GenerateCmd = &cobra.Command{
```

Comment:
Instead of making the command public, I would suggest using rootCmd.Find method to get the command in the cmd.go switch case:

// Retrieve the 'second' command from the root command
generateCommand, _, err := rootCmd.Find([]string{"generate"})
if err != nil {
    fmt.Println("Error finding 'generate' command:", err)
    return
}

// Execute the 'generate' command
if err := generateCommand.Execute(); err != nil {
    fmt.Println("Error executing 'genereate' command:", err)
}